pkg/kubelet/userns/userns_manager_test.go

```diff
@@ -51,6 +51,10 @@ func (m *testUserNsPodsManager) ListPodsFromDisk() ([]types.UID, error) {
 	return m.podList, nil
 }
 
+func (m *testUserNsPodsManager) HandlerSupportsUserNamespaces(runtimeHandler string) (bool, error) {
+	return true, nil
```

Comment:
Unrelated to these lines, but can we add some tests for the new changes in `GetOrCreateUserNamespaceMappings()`?

Reply:
Lot of test added now